src/components/Footer/Legal.ts

```diff
@@ -0,0 +1,170 @@
+@localized()
+export class Legal extends LitElement {
+  static styles = [
+    theme,
+    contentStyles,
+    css`
+      .table-container {
+        overflow-x: auto;
+        margin-bottom: 0.9375rem;
+      }
+
+      table {
+        width: 100%;
+        border-collapse: collapse;
+        border-spacing: 0;
+      }
+
+      tbody,
+      tfoot,
+      thead {
+        border: none;
+        background-color: transparent;
+      }
+
+      tfoot td,
+      tfoot th,
+      thead td,
+      thead th,
+      tbody td,
+      tbody th {
+        padding: 0.75rem;
+        text-align: left;
+      }
+
+      th,
+      th {
+        font-weight: 1rem;
+        font-weight: 500;
+      }
+
+      tbody td {
+        font-size: 0.875rem;
+        line-height: 1.125rem;
+        border-bottom: 1px solid var(--bkd-table-border);
+      }
+
+      thead,
+      tbody tr:last-of-type th,
+      tbody tr:last-of-type td {
+        border-bottom: 1px solid var(--bkd-brand-black);
+      }
+    `,
+  ];
+
+  render() {
+    return html`
+      <h2>${msg("Haftungsausschluss")}</h2>
+      <p>
+        ${msg(
+          "Die Inhalte dieser Fachapplikation sind ein Datenangebot der zuständigen Schule. Obwohl diese die Daten regelmässig auf Richtigkeit und Aktualität hin prüft, lehnen wir, soweit gesetzlich zulässig, jede Haftung für unerwünschte Folgen aus dem Gebrauch dieser Daten ab.",
+        )}
+      </p>
+      <p>
+        ${msg(
+          "Diese Fachapplikation enthält Links zu Angeboten von Dritten. Sie benutzen diese Links auf eigenes Risiko. Für die Inhalte dieser anderen Webseiten übernehmen wir keine Verantwortung. Von unserer Haftung ausgeschlossen ist namentlich die Garantie für die ständige Verfügbarkeit unserer Fachapplikation sowie Schäden (z.B. wegen Verbindungsunterbruch oder schädlicher Software), die durch die Benutzung dieser Fachapplikation entstehen könnten.",
+        )}
+      </p>
+
+      <h2>${msg("Immaterialgüterrechte")}</h2>
+      <p>
+        ${msg(
+          "Sichtbare Elemente (Bilder/Icons) dieser Fachapplikation können Dritten gehören. Diese Elemente dürfen deshalb grundsätzlich nicht verwendet werden. Der Kanton Bern kann deren Verwendung ausserhalb der Webseite auf Anfrage erlauben, sofern er dazu selbst berechtigt ist. Für solche Anfragen wenden Sie sich bitte an die Stelle, die für die Fachapplikation verantwortlich ist. Die Kontaktangaben finden Sie im Impressum.",
+        )}
+      </p>
+
+      <h2>${msg("Datenschutzerklärung")}</h2>
+      <p>
+        ${msg(
+          "Verantwortliche Stelle im Sinne der Datenschutzgesetze, unter anderem auch der EU-Datenschutzgrundverordnung (DSGVO), ist:",
+        )}
+      </p>
+      <p>
+        ${withLineBreaks(
+          msg(`Mittelschul- und Berufsbildungsamt
+Kasernenstrasse 27
+3013 Bern`),
+        )}
+      </p>
+      <p><a href="mailto:fbi.mba@be.ch">${msg("E-Mail")}</a></p>
+      <p><a href="https://www.be.ch/mba">www.be.ch/mba</a></p>
```

Comment:
In der französischen Version ist der Link anders in der Vorlage: [www.be.ch/omp](http://www.be.ch/omp)

Reply:
:pray: Danke, sehr aufmerksam, ich nehm' das noch mit in die Übersetzung auf.